_inc/client/search/index.jsx

```diff
@@ -105,6 +105,7 @@ export const SearchResults = ( {
 		let isPro = 'scan' === element[0]
 				 || 'akismet' === element[0]
 				 || 'backups' === element[0]
+				 || 'videopress' === element[0]
```

Comment:
Sam has a good point that it's easier to test it if we have it based on `branch-4.4`. We'll squash the end result and cherry-pick it over to `master`.